specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/managedClusters.json

```diff
@@ -2985,6 +3080,56 @@
           "description": "The private link service connection description."
         }
       }
+    },
+    "PrivateLinkResourcesListResult": {
+      "type": "object",
+      "description": "A list of private link resources",
+      "properties": {
+        "value": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
```

Comment:
could you pls use https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/privatelinks.json#L99 ? We're trying to use common definition for privatelink cross services for consistency. cc @allenjzhang .

Reply:
we can't use the common definition for this currently, there are some differences, like 'readonly' field. 
Had talked this to Catherine offline, she is okay with it.